app/assets/javascripts/training/components/draggableSlide.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useBlockDrag } from '../../hooks/useBlockDrag';
+
+const DraggableSlide = ({
+  slide,
+  heading,
+  description,
+  canDrag,
+  animating,
+  onDrag,
+  canSlideMoveUp,
+  canSlideMoveDown,
+  onSlideMoveUp,
+  onSlideMoveDown,
+  index
+  }) => {
+  const [isHoverArrowDown, setIsHoverArrowDown] = useState(false);
+  const [isHoverArrowUp, setIsHoverArrowUp] = useState(false);
+  const { ref, isDragging, handlerId } = useBlockDrag({
+    block: slide,
+    canDrag,
+    isAnimating: animating,
+    onBlockDragOver: onDrag
+  });
+
+  const opacity = isDragging ? 0.5 : 1;
+
+  return (
+    <div key={heading} className={'program-description'} style={{ opacity }} ref={ref} data-handler-id={handlerId}>
+      <div className="draggable-slide-container">
+        <div className="program-description__header">
+          <h4><strong>{heading}</strong></h4>
+          {description.split('\n').map((paragraph, i) => paragraph && <p key={i}>{paragraph}</p>)}
```

Comment:
This is where it errors when I enter reorder mode for a yml-based module. `description` comes from `slide.wiki_page`, so when the wiki_page is null, it will error.

Reply:
I added a null check there.